example.py

```diff
@@ -21,8 +21,8 @@
 engine = create_engine(
     "rockset_sqlalchemy://",
     connect_args={
-        "api_server": os.environ["ROCKSET_API_SERVER"],
-        "api_key": os.environ["ROCKSET_API_KEY"],
+        "host": os.environ["ROCKSET_API_SERVER"],
+        "username": os.environ["ROCKSET_API_KEY"],
```

Comment:
Seems unintuitive that apikeys are passed in as usernames. Apikeys are secrets while usernames aren't necessarily.  How does this get passed down the Connection object?

Reply:
I've reverted this back to the behavior of v0 in https://github.com/rockset/rockset_sqlalchemy/pull/2/commits/3e25c95af5e51a089be9ea11ea1d9cdf74dfe1c2